recipe/bld.bat

```diff
@@ -92,6 +92,9 @@ if errorlevel 1 exit 1
 copy *.lib %LIBRARY_LIB%\ || exit 1
 if errorlevel 1 exit 1
 
+# libgdal 2.3.0
+copy %SRC_DIR%\port\cpl_vsi_error.h %LIBRARY_PREFIX%
```

Comment:
Should this be `%LIBRARY_INC%` instead?

Reply:
Sorry, copy and pasta (this is what I get for sending PRs late at night :smile:)